portfolio/src/main/webapp/index.html

```diff
@@ -62,6 +62,25 @@ <h1>My Portfolio</h1>
         <p>Click here to get a random fact about me:</p>
         <button onclick="addRandomFact()">Fact!</button>
         <div id="fact-container"></div>
+        <form
+            action="/data" 
+            id="comment-form"
+            method="POST">
+          <p>Submit a comment here:</p>
+          <br>
+          <textarea
+              id="comment-input-field"
+              name="comment-input-field"></textarea> <!-- Prevent whitespace from rendering -->
```

Comment:
sorry, what do you mean by the comment  "Prevent whitespace from rendering"?

Reply:
If I follow the style guide and put </textarea> on the next line, the whitespace between the <textarea> opening tag and </textarea> closing tag will be rendered inside the text area box on the website (i.e. there will be some extra spaces before the user's comment). Probably handled if I include trim in the getComment method below.

Reply:
right, inside text area tag should be empty.